New Rule: Apply arithmetic operations on non-nullables only (#51)
Qowaiv/qowaiv-analyzers: rules/QW0017.md

﻿# QW0017: Apply arithmetic operations on non-nullables only 

.NET allows arithmetic operations between nullable value types. The outcome of

> those operations will potentially by `null` if any of the operants has no value
> runtime. This behavior is commonly undesirable, and occurrences are not trivial
> to spot, potentially leading to bugs.

Reviewer: ```suggestion
those operations will potentially be `null` if any of the operants has no value
at runtime. This behavior is usually undesirable, and occurrences are not trivial
to spot, potentially leading to bugs.
```
